core/src/Instance/B2CAuthority.cs

```diff
@@ -37,37 +38,85 @@ internal class B2CAuthority : AadAuthority
     {
         public const string Prefix = "tfp"; // The http path of B2C authority looks like "/tfp/<your_tenant_name>/..."
         public const string B2CCanonicalAuthorityTemplate = "https://{0}/{1}/{2}/{3}/";
+        public const string MicrosoftOnline = "https://login.microsoftonline.com";
+        public const string OpenIdConfigurationEndpoint = "v2.0/.well-known/openid-configuration";
+        internal const string B2CTrustedHost = "b2clogin.com";
```

Comment:
All the other constants are public, keep this one public as well. It doesn't really matter if a member is public or internal if the class is internal, so consistecy trumps. I'd recommend using public inside internal classes.

Reply:
this was copy/paste, so thanks for picking it up. 

---
In reply to: [230918925](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1383#discussion_r230918925) [](ancestors = 230918925)